docs/api.md

```diff
@@ -18083,6 +18083,13 @@ TargetAllocator indicates a value which determines whether to spawn a target all
           Resources to set on the OpenTelemetryTargetAllocator containers.<br/>
         </td>
         <td>false</td>
+      </tr><tr>
+        <td><b><a href="#opentelemetrycollectorspectargetallocatorsecuritycontext">securityContext</a></b></td>
+        <td>object</td>
+        <td>
+          TopologySpreadConstraints embedded kubernetes pod configuration option, controls how pods are spread across your cluster among failure-domains such as regions, zones, nodes, and other user-defined top<br/>
```

Comment:
Could you regenerate the documentation as well? We still have the `TopologySpreadConstraints` description here.